package.json

```diff
@@ -71,7 +71,7 @@
     "universal-cookie": "4.0.4"
   },
   "peerDependencies": {
-    "@edx/paragon": "^12.8.0",
+    "@edx/paragon": ">=12.8.0 <14.0.0",
```

Comment:
I believe we could get away with `>=10.0.0 <14.0.0` as the Paragon upgrade in this library only needs when we first introduced the react-bootstrap passthru for the `Button` component, which was introduced on Paragon@10.0.0.